core-common/src/main/resources/event-type-schema/_metadata/_metadata.3.avsc

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "Metadata",
+  "type": "record",
+  "namespace": "org.zalando.nakadi.generated.avro",
+  "doc": "Event metadata defines data is not related to actual event payload data",
```

Comment:
maybe we can say instead to what that date relates? like `Event metadata defines data about the payload and additional information for Nakadi operations`

Reply:
Alright that sounds better, updated the same